dailystats.py

```diff
@@ -17,13 +17,11 @@
 LOGGER = logging.getLogger(__name__)
 DEBUG = False
 LOGGER.setLevel(logging.DEBUG if DEBUG else logging.INFO)
-# TODO(Jack): Remove knowledge of platform/filesystem.
-INPUT_FILE_NAME = "C:\\Geraldine\\Coronavirus\\coronavirus-cases20200427.csv"
-OUTPUT_FILE_NAME = "C:\\Geraldine\\Coronavirus\\coronavirus-output_results.csv"
+INPUT_FILE_NAME = "coronavirus-cases20200427.csv"
+OUTPUT_FILE_NAME = "coronavirus-output_results.csv"

```

Comment:
Relative paths seem to be sufficient since you are reading and writing from the directory this runs in!

However, the best way to construct paths is 
```
import os

PATH = os.path.join("foo", "bar", "baz")
```

As this will deal with windows vs unix